docs/ehrql-new-tutorial-intro.md

```diff
@@ -81,76 +54,71 @@ Familiarity with:
 would also be helpful,
 but is *not* essential.
 
-#### Software requirements
+## How to work through the Tutorial Series
+The tutorial is split into 7 sections that we recommend
+you work through sequentially. Each section builds on previous knowledge. Throughout the tutorial series, you will see optional explanations and reading, that you might 
+want to explore at the time or afterwards.
 
-To try running or modifying the examples in this tutorial,
-you will need to install Data Builder.
-
-##### Operating system
+We recommend that you work through the tutorial by installing Data Builder, downloading the sample code, and 
+running the code on your machine. 
 
-Data Builder runs on Windows, macOS, and Linux either via:
-
-* the OpenSAFELY CLI
-* a manually installed Python package
-
-##### Text editor
-
-You can edit dataset definitions in any text editor.
-
-You may find it useful to use a text editor or integrated development environment
-that supports Python syntax.
+### Learning objectives
+This tutorial series will introduce you to writing and running 
+ehrQL queries with Data Builder. 
 
-If you have no preference already,
-[Visual Studio Code](https://code.visualstudio.com) is a reasonable, free-of-charge choice
-that runs on Windows, macOS and Linux.
+By the end of the tutorials, you will be able:
 
-!!! tip
-    Because dataset definitions are Python files,
-    you can use auxiliary developer tools to keep the files tidy.
+* to install Data Builder
+* to write a dataset definition in ehrQL
+* to run a dataset definition with Data Builder
+* to use electronic health record codelists in ehrQL
+* to perform queries via selecting, filtering and aggregation
 
-    For instance, the dataset definitions in this tutorial are checked with the following tools:
+Each tutorial section will have its own learning objectives. 
 
-    * [black](https://github.com/psf/black), to format dataset definitions automatically
-    * [flake8](https://github.com/PyCQA/flake8), to suggest coding style improvements
-    * [isort](https://github.com/PyCQA/isort), to order `import` statements consistently
```

Comment:
thought: In lieu of having an `opensafely format` command, it might still be useful to have a separate "Guidelines for formatting/organising dataset definitions" page which includes those suggestions and states that the dataset definitions included in the documentation use those tools.

Reply:
I am not sure. But could discuss further. My feeling is that people will not use these tools unless there is an easy way to access them.